src/BackerymailsEntityAccessControlHandler.php

```diff
@@ -24,7 +24,7 @@ protected function checkAccess(EntityInterface $entity, $operation, AccountInter
         return AccessResult::allowedIfHasPermission($account, 'administer backerymails');
```

Comment:
Add a Kernel tests for this new permission. You can see an example of this kind of tests here:
https://www.drupal.org/files/issues/2018-12-11/2809291-58.patch